src/helpers/eventBridge.js

```diff
@@ -99,6 +106,19 @@ export default class EventBridge {
 
     const result = await this.sqsClient.receiveMessage(queueParams).promise();
 
+    const messageHandlers = result.Messages.map(
```

Comment:
Will this cause issues with tests run in parallel?

> Tests are parallelized by running them in their own processes to maximize performance
https://jestjs.io/

Reply:
This is why for now at least the `--runInBand` option is needed.
While we have a single SQS per EventBridge this will be the case.